packages/presentation/lib/utils/responsive.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+enum ScreenType {
+  mobile,
+  desktop,
+}
+
+class Responsive {
+  static ScreenType getScreenType(BuildContext context) {
+    final width = MediaQuery.of(context).size.width;
+
+    if (width < 500) {
+      return ScreenType.mobile;
+    }
+
+    return ScreenType.desktop;
```

Comment:
nice check from the side of responsive design, but not right from the logical side. I think that u should check if current platfrom Android/iOS => mobile. else desktop

Reply:
@sergei-mikhailovskii-idf under mobile and desktop I mean sizes not devices, Android can be presented on television.